Fix disconnecting on TURN relay session (#198)
planetarium/libplanet: CHANGES.md

   round-trips.  [[#187], [#190]]
 - Improved the read throughput of `BlockChain<T>` while mining through
   `BlockChain<T>.MineBlock()`
 - Improved the read throughput of `BlockChain<T>` when calling
   `BlockChain<T>.Append()`.

>  - Fixed a bug that `Swarm `attempted to use TURN relay even though `_host` was

Reviewer: ```suggestion
 - Fixed a bug that `Swarm` attempted to use TURN relay even though `_host` was
```

    and its all implementations.  [[#189]]
 -  The signature of `IStore.PutBlock<T>(Block<T>)` method was changed to
    `PutBlock<T>(Block<T>, Address)`.  [[#189]]

>  -  Improved the read throughput of `BlockChain<T>` when calling

Reviewer 2: ```suggestion
 -  Improved the read throughput of
```

    `BlockChain<T>.Append()`.

>  -  Fixed a bug that `Swarm` attempted to use TURN relay even though `_host` was

Reviewer 2: As the `_host` field is just internal details that library users are not aware of, we'd better to refer to the `Swarm` contstructor's `host` parameter instead:

```suggestion
 -  Fixed a bug that `Swarm` had attempted to use TURN relay even if the `host` argument was
```

    given.

>  -  Fixed a bug that TURN relay was disconnected when connecting for more than 5

Reviewer 2: ```suggestion
 -  Fixed a bug that TURN relay had been disconnected when being connected for longer than 5
```
